VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -84,21 +80,21 @@ context('VAModalPicker', () => {
   describe('when an option is selected', () => {
     it('should update selected to the value of that option and select done', async () => {
       await waitFor(() => {
-        selectionButtons[0].props.onPress()
+        fireEvent.press(component.getByTestId('Java'))
       })
 
       await waitFor(() => {
-        doneButton.props.onPress()
+        fireEvent.press(doneButton)
       })
 
       expect(selected).toEqual('java')
     })
 
     it('should not update selected to the value of that option and select cancel', async () => {
       await waitFor(() => {
-        selectionButtons[0].props.onPress()
+        fireEvent.press(component.getByTestId('Java'))
```

Comment:
same thing as above

Reply:
getByRole('menuitem', { name: "Java" }) worked, but for some reason it was still picking up the "java" from before so I changed this one to "JavaScript2" and it works now.